src/components/Wowerlay.tsx

```diff
@@ -8,85 +7,106 @@ import {
+        // If we use transform: true, animation that uses transform property will be broken.
+        transform: false,
+        whileElementsMounted(target, popover, update) {
+          if (props.fixed) {
+            update();
+            return NOOP;
+          }
 
-    expose({
-      update() {
-        wowerlayInstance.value?.update();
+          return autoUpdate(target, popover, update, {
+            ancestorResize: true,
+            ancestorScroll: true,
+            elementResize: true,
+          });
+        },
+        middleware: computed<Middleware[]>(() => {
+          const middlewares = [attrs()] as Middleware[];
+
+          if (typeof props.gap === 'number' && props.gap !== 0) middlewares.push(offset(props.gap));
+          if (!props.noFlip) middlewares.push(flip());
+          if (props.syncSize) middlewares.push(syncSize());
+          if (!props.canLeaveViewport) middlewares.push(shift({ crossAxis: true }));
+
+          return middlewares;
+        }),
       },
-    } as WowerlayTemplateRef);
+    );
 
-    const tooltipIsClosable = ref(false);
-    const tooltipIsVisible = computed(() => isElement(props.target) && props.visible);
+    watch(popoverEl, (el) => {
+      emit('update:el', el);
+    });
+
+    const popoverClosable = ref(false);
+    const popoverVisible = computed(() => isElement(props.target) && props.visible);
 
     const close = () => {
       if (!props.visible) return;
 
-      if (tooltipIsClosable.value) {
+      if (popoverClosable.value) {
         emit('update:visible', false);
       }
     };
 
     onBeforeUnmount(close);
 
-    const backgroundEl = shallowRef<HTMLElement>();
     const handleWindowClick = (e: MouseEvent) => {
-      // We should not call this function if Wowerlay is not visible
-      // or has no target element etc.
       if (
         !props.visible ||
         !(props.target instanceof HTMLElement) ||
-        // This check is for TypeScript, TypeScript doesn't think e.target is HTMLElement
         !(e.target instanceof HTMLElement) ||
-        // This simulates `stopPropagation` but do not block event bubbling
-        e.target.closest(STOP_ATTR_QUERY)
+        // Some people (like us in Wope) don't want to use `event.stopPropagation()` to prevent Wowerlay closing
+        // Because it blocks following analytics or other events that rely on click.
+        // If clicked element or it's ancestors has this attribute, Wowerlay doesn't close.
+        e.target.closest('[data-wowerlay-stop]')
       ) {
         return;
       }
 
-      const scopeEl = e.target.closest(SCOPE_ATTR_QUERY);
-
-      // If scope element exists but it isn't our Wowerlay's scope we just return.
+      // If clicked element or any ancestors has scope attribute we only
+      // close popovers attached to elements in the target scope.
+      // @See Demo/Nested and click the child Wowerlay body, it won't close parent Wowerlay
+      // because each Wowerlay popover is a new scope.
```

Comment:
"@See" isn't meant to be used with descriptions, maybe we can provide a link to the demo to try the feature